chardet/langgreekmodel.py

```diff
@@ -25,201 +25,192 @@
 # 02110-1301  USA
 ######################### END LICENSE BLOCK #########################
 
-# 255: Control characters that usually does not exist in any text
-# 254: Carriage/Return
-# 253: symbol (punctuation) that does not belong to word
-# 252: 0 - 9
+Windows_1253_Greek_char_to_order_map = (
```

Comment:
It is due to automatic script 'create_language_model.py' for langmodel building/creation, which I've created. The name of the charmap tuple is automatically taken from the 'CharsetsTabs.txt' table. 
I am too lazy to keep/rewrite all charmap names (for all language models) to the "old names". 
Please look at the my new 'sbcsgroupprober.py'. 
Btw: It is only a name like name of a variable. I think it can be any regular name :)